src/App.jsx

```diff
@@ -42,11 +42,15 @@ class App extends Component {
           endTime: Date.now(),
         }}
         render={({ error, props }) => {
+          console.log(props);
           if (error) {
             return <div>{error.message}</div>;
+          } else if (props && typeof props !== 'undefined') {
+            return <Map trynState={props} />;
           }
-          return <Map trynState={props || null} />;
-        }}
+          return <div>Loading...</div>;
```

Comment:
I believe this overwrote changes I made: https://github.com/trynmaps/opentransit-map/issues/45
E.g. remove loading text and show the map page by default